installer/darwin/create_dmg.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+# Create a read-only disk image of the contents of a folder
+# This script is derived from the create-dmg utility
+# https://github.com/create-dmg/create-dmg
```

Comment:
Derived from this file: https://github.com/create-dmg/create-dmg/blob/master/create-dmg
Just stripped it down to the options that we use.